packages/flutter_test/lib/src/controller.dart

```diff
@@ -817,26 +817,32 @@ abstract class WidgetController {
   /// Shorthand for `Scrollable.ensureVisible(element(finder))`
   Future<void> ensureVisible(Finder finder) => Scrollable.ensureVisible(element(finder));
 
-  /// Repeatedly scrolls the `scrollable` by `delta` in the
+  /// Repeatedly scrolls a [Scrollable] by `delta` in the
   /// [Scrpllable.axisDirection] until `finder` is visible.
```

Comment:
Typo of the old code `Scrpllable` -> `Scrollable`.

Reply:
Done.